app/views/subscription_mailer/send_digest.html.erb

```diff
@@ -20,7 +20,8 @@
           </div>
           <div style="float: left;">
             <b style="font-weight: 500;"><%= n.author.username.capitalize %></b>
-            <p style="margin: 0;color: #999;font-weight: 500;">Published Feb 2</p>
+            <p style="margin: 0;color: #999;font-weight: 500;">Published <%= Time.now.strftime("%m/%d") %>
```

Comment:
Hi, it looks like you may need to use, instead, `n.created_at.strftime("%m/%d")` - to show the time of publication of the content, rather than the time the email is sent?

Reply:
Yes exactly, that's what I was thinking while trying this locally! @arungoel123456 kindly make this change and I think we'll be good to go. Thanks a lot!